corehq/messaging/scheduling/filters.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import
+from corehq.apps.data_interfaces.models import AutomaticUpdateRule
+from corehq.apps.reports.filters.base import BaseReportFilter
+from django.utils.translation import ugettext_lazy
+
+
+class ScheduleInstanceFilter(BaseReportFilter):
+    label = ugettext_lazy("")
```

Comment:
@gcapalbo Is this just an omission or is the label supposed to be empty?  If it's supposed to be empty, we could probably just drop the `ugettext_lazy`.  I'm getting an error regarding this when I update translations.
```
./corehq/messaging/scheduling/filters.py:9: warning: Empty msgid.  It is reserved by GNU gettext:
                                                     gettext("") returns the header entry with
                                                     meta information, not the empty string.
```

Reply:
@esoergel it ends up not being used because it uses a custom template, but the base class requires it to be defined if i remember. fine to just drop `ugettext_lazy`

Reply:
perfect.  Here it is: https://github.com/dimagi/commcare-hq/pull/20312